File: Python/ex21.py
# Uma escola com cursos em regime semestral realiza duas avaliações durante o semestre e calcula 
# a média do aluno, da seguinte maneira:
#   "MEDIA = (P1 + 2P2) / 3"
#Fazer um programa para entrar via teclado com o valor da primeira nota (P1) e o programa 
# deverá calcular e exibir quanto o aluno precisa tirar na segunda nota (P2) para ser aprovado, 
# sabendo que a média de aprovação é igual a cinco.
media = 5
p2 = float

p1 = float(input("Digite a nota da P1: "))

p2 = ((3*media)/2)-p1 

print(f"Você precisa tirar {p2:.2f} na P2 para ser aprovado!")

File: C/ex21.c
//exercicio 21

#include <stdio.h>
#include <stdlib.h>

int main(int argc, char const *argv[])
{
    // Declaração de variáveis
  float p1, p2;

  // Solicita entrada de dados para o usuário e grava o valor digitado em uma variável "p1"
  printf("Digite a nota P1: ");
  scanf("%f", &p1); 
  
  p2=(15-p1)/2;

  printf("O aluno precisa tirar no minimo %.2f na P2 para ser aprovado ",p2);

    return 0;
}
